### ISSUE TITLE
lol

### ISSUE THREAD

**Issue by something-0001** [syslinks/Anonymous-Stealer#1]
![image](https://github.com/syslinks/Anonymous-Stealer/assets/14898342/61b587c8-ff3e-4260-96a9-a4f0f606739f)

**Comment by thecrazytech1** [comment]
The file literally has nothing in it though

**Comment by iloveyumyumshrimpnoodles** [comment]
> The file literally has nothing in it though, like I went to the github repo that it has there and there is literally nothing in the file

yeah seems 100% empty
![image](https://github.com/syslinks/Anonymous-Stealer/assets/125209101/ded7d193-5abb-49e9-8c8c-1bacaacc7ce5)

**Comment by something-0001** [comment]
I deal with too many retards to count lmfao

**Comment by iloveyumyumshrimpnoodles** [comment]
yeah, it's crazy how many fucktards try backdooring copied stealers, and then do a shitty job aswell

**Comment by thecrazytech1** [comment]
> 

I'm not even a contributer to this repo anyway, so idrc (and that's not the file I looked at, I looked at the file that the person has sent a pic of, that's what I was refering too. Just that file nothing else)

**Comment by patriotic69** [comment]
hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh
